fitbenchmarking/templates/table_template.html

```diff
@@ -139,44 +139,48 @@ <h2 class="handle">
                             <label for="handle3">Performance profile</label>
                             </h2>
                             <div class="content">
-                                <div class="graph">
+                                <div>
                                     <p> For a more interative plot (Dash), switch to "Online version" by clicking the button below.
                                         <b>This will only work if the Dash app is running.</b> </p>
                                     {% for offline_plot, dash_plot, index in zipped_paths %}
+                                        <br>
                                         <div>
-                                        <button class="btn default" id="offline_plot{{ index }}"
-                                                data-value1="{{ offline_plot }}" data-value2="{{ index }}"
-                                                onclick="load_src(this)">Offline version</button>
-
-                                        <button class="btn default" id="online_plot{{ index }}"
-                                                data-value1="{{ dash_plot }}" data-value2="{{ index }}"
-                                                onclick="load_src(this)">Online version</button>
-                                        </div>
+                                            <div>
```

Comment:
Is this `<div>` spare?

Reply:
I updated this part of code now that I rebased

Reply:
Also, the change to pylint settings is for the changes in tables.py , where I'm adding the variable n_solvers_large around line 122 (so that means 31 local variables rather than 30). Not sure whether there's another way around that